openfisca_us/variables/gov/states/md/tax/income/deductions/md_standard_deduction.py

```diff
@@ -29,4 +29,4 @@ def formula(tax_unit, period, parameters):
             (filing_status == filing_statuses.SINGLE)
             | (filing_status == filing_statuses.SEPARATE),
         )
-        return where(is_single_separate, single_separate, joint_head_widow)
+        return where(is_single_separate, single_separate, joint_head_widow)[0]
```

Comment:
@nikhilwoodruff any idea why this would be returning a two-dimensional array? Without the `[0]`, other computations break.

I also wonder if this might be related to the recursion error #1134.

Reply:
Oh, it's because of that comma at the end of line 30. `is_single_separate` looks like this when debugging:
```python
(np.array([True]),)
```

Reply:
Fixed now.